### PR TITLE
More cardano-config module movement to cardano-node

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Byron/Query.hs [hunk 1]
```diff
@@ -25,7 +25,8 @@ import           Cardano.Config.Types (SocketPath (..))
 import           Cardano.Api.LocalChainSync (getLocalTip)
 import           Cardano.CLI.Environment
                    (EnvSocketError, readEnvSocketPath, renderEnvSocketError)
-import           Cardano.TracingOrphanInstances.HardFork ()
+-- TODO @intricate: This forces us to import "cardano-node". Fix this.
+import           Cardano.Tracing.OrphanInstances.HardFork ()
```
Comment (intricate) [inline cardano-cli/src/Cardano/CLI/Byron/Query.hs:29]: I need to address this.

Comment (Jimbo4350) [reply cardano-cli/src/Cardano/CLI/Byron/Query.hs:29]: Lets address this after this PR is merged.

Comment (intricate) [reply cardano-cli/src/Cardano/CLI/Byron/Query.hs:29]: Alright, well just remove the "TODO @intricate".

```suggestion
-- TODO: This forces us to import "cardano-node". Fix this.
import           Cardano.Tracing.OrphanInstances.HardFork ()
```

Comment (Jimbo4350) [reply cardano-cli/src/Cardano/CLI/Byron/Query.hs:29]: I'll fix this in my next PR, I've been fighting with CI for some time.

**Pull request merged**